index.ts

```diff
@@ -132,7 +132,7 @@ async function sendEventToIntercom(
                 event_name: event,
                 created_at: eventSendTime,
                 email,
-                id: distinct_id,
```

Comment:
⚠️ Intercom seems to [require their Contact model's primary key here](https://developers.intercom.com/intercom-api-reference/reference/submit-a-data-event).

I haven't been able to confirm quite yet if [this user_id/id property](https://developers.intercom.com/intercom-api-reference/reference/the-data-event-model#:~:text=Your%20identifier%20for%20a%20lead%20or%20a%20user.), "Your identifier for a lead or a user",  is the [Contact.user_id/id](https://developers.intercom.com/intercom-api-reference/reference/the-contact-model) or [Contact.external_id](https://developers.intercom.com/intercom-api-reference/reference/the-contact-model) 🤔